site/src/lib/api/wrangler/index.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  async fetch(request, env) {
```

Comment:
We need these wranglers because we want objects in the bucket to be public, and it is not possible to just set that for the bucket.